### DISCUSSION TITLE
Welcome to RyzenAdj Discussions!

### DISCUSSION THREAD

**Discussion by FlyGoat** [FlyGoat/RyzenAdj#262] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 900Ninety** [comment]
Hi 

Not sure why , getting these errors 
![image](https://github.com/FlyGoat/RyzenAdj/assets/99098206/f4d00640-1cbc-43c7-bbfb-7e14be80319f)









am i missing something?

**Comment by spariant** [comment]
Hi Flygoat,
I'm here to troubleshoot the power limits on my mobile Ryzen APU minipc.

Thanks for a great tool!
